server/nimble_server.py

```diff
@@ -19,22 +19,41 @@
 # Allow imports from the parent directory (potential security issue)
 sys.path.append(os.path.dirname(__file__) + "/..")
 
-from generate import generate_docs
+from generate import generate
 
 app = FastAPI()
 
 # Static files for the test page
 app.mount("/static", StaticFiles(directory="static"), name="static")
 
 
-# http://127.0.0.1:8000/wakoma/nimble/test_page
-@app.get("/wakoma/nimble/test_page")
+# http://127.0.0.1:8000/wakoma/nimble/configurator
+@app.get("/wakoma/nimble/configurator")
 async def read_index():
     """
     Loads a sample page so that the system can be tested end-to-end.
     """
 
-    return FileResponse('test_page.html')
+    return FileResponse('configurator.html')
+
+
+def get_unique_name(config):
```

Comment:
I like the term unique_name. I think it is clear and unambiguous.

Reply:
Do we want to make an issue to move to this nomenclature? Is it replacing, the "hash" terminology?

Reply:
The references to configuration hashes have already been removed from orchestration, as we stopped hashing there. It may be easier to change the four instances in `index.js` as part of this PR?